doc/workshops/plugin-development.rst

```diff
@@ -2,7 +2,7 @@
 Plugin Development with Spyder
 ##############################
 
-This workshop consists of a review of the features and possibilities of the API offered by `Spyder`_ 5, the recently released version of our favorite IDE for Scientific Python, to develop plugins to extend its functionality.
+This workshop consists of a review of the features and possibilities of the API offered by `Spyder`_ 5, the recently released version of our favorite IDE for Scientific Python, to develop plugins and extend its functionality.
```

Comment:
```suggestion
This workshop reviews the features and possibilities of the API offered by `Spyder`_ 5—the recently released version of our favorite IDE for scientific Python—for plugin development and to extend its functionality.
```

Tweaked the syntax and phrasing a bit to be a little simpler and easier to understand, and made "scientific" in "scientific Python" lowercase, per convention.